site/content/docs/0.0/migration.md

```diff
@@ -158,8 +194,31 @@ toc: true
   - **Tallest**: `.gap-scaled-tallest`, `.row-gap-scaled-tallest`, `.column-gap-scaled-tallest`
   - **Spacious**: `.gap-scaled-spacious`, `.row-gap-scaled-spacious`, `.column-gap-scaled-spacious`
 
+#### Text
+
+- <span class="badge text-bg-success">New</span> Alignment text utilities: `.text-start`, `.text-center`, `.text-end`, and responsive versions like `.text-{breakpoint}-{start|end|none}` where `breakpoint` is in `|xs|sm|md|lg|xl|2xl|3xl`.
+
+- <span class="badge text-bg-success">New</span> Wrapping and overflow text utilities: `.text-wrap` and `.text-nowrap`.
+
+- <span class="badge text-bg-success">New</span> Word break text utility: `.text-break`.
+
+- <span class="badge text-bg-success">New</span> Text transform text utilities: `.text-lowercase`, `.text-uppercase`, and `.text-capitalize`.
+
+- <span class="badge text-bg-success">New</span> Font size text utilities:
+  - **Display headings**: `.fs-dl`, `.fs-dm`, `.fs-ds`
+  - **Headings**: `.fs-hxl`, `.fs-hl`, `.fs-hm`, `.fs-hs`
+  - **Body**: `.fs-bl`, `.fs-bm`, `.fs-bs`
+  - **Labels**: `.fs-lxl`, `.fs-ll`, `.fs-lm`, `.fs-ls`
```

Comment:
Not quite sure that we should include this in the migration guides, and more braodly in the doc tbh, the labels should only be used inside components. Maybe we can remove the references in here or maybe explain it more in the doc and here that they should only be used in some very specific contexts ?

Reply:
Hum good point. I would even ask if we should generate them if they should not be used ?
The mixin to generate them would be useful, but the corresponding utilities, will/should never be used.

(It's not very handy since utilities and size handled by the mixin are in the same map)

Reply:
I removed these while keeping it available in the font mixin.